code/components/extra-natives-five/src/InterfaceExtraNatives.cpp

```diff
@@ -43,13 +152,42 @@ static char LoadZoomMapDataMeta()
 	return success;
 }
 
+static void PatchResetHudComponentValues()
+{
+	static constexpr uint64_t nativeHash = 0x450930E616475D0D;
+
+	auto handler = fx::ScriptEngine::GetNativeHandler(nativeHash);
+
+	if (!handler)
+	{
+		trace("Couldn't find 0x%08x handler to hook!\n", nativeHash);
+		return;
+	}
+
+	fx::ScriptEngine::RegisterNativeHandler(nativeHash, [=](fx::ScriptContext& ctx)
+	{
+		auto index = ctx.GetArgument<uint32_t>(0);
+
+		if (auto& it = hudComponentOldEntries.find(index); it != hudComponentOldEntries.end())
+		{
+			auto component = &hudComponentsArray[index];
+			component->alignX[0] = it->second.alignX[0];
```

Comment:
This partial reset code seems to be in *two* places, maybe it could be unified (also in case any other editable values get added in the future?).